app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,8 @@
 
  */
 
+ @import 'modules/sitter';
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Prefer double-quoted strings